src/pygambit/game.pxi

```diff
@@ -336,6 +337,39 @@ class Game:
         g.title = title
         return g
 
+    def to_arrays(self, dtype: typing.Type = Rational) -> typing.List[np.array]:
+        """Generate the payoff tables for players represented as numpy arrays.
+
+        Parameters
+        ----------
+        dtype : type
+            The type to which payoff values will be converted and
+            the resulting arrays will be of that dtype
+
+        Returns
+        -------
+        list of np.array
+
+        See Also
+        --------
+        from_arrays : Create game from list-like of array-like
+        """
+        arrays = []
+
+        shape = tuple(len(player.strategies) for player in self.players)
+        for player in self.players:
+            array = np.zeros(shape=shape, dtype=object)
+            for profile in itertools.product(*(range(s) for s in shape)):
+                try:
+                    array[profile] = dtype(self[profile][player])
+                except (ValueError, TypeError, IndexError, KeyError):
+                    raise ValueError(
+                        f"Payoff '{self[profile][player]}' cannot be \
```

Comment:
Splitting strings like this doesn't quite do what you're probably wanting:

```
File ~/git/gambit/src/pygambit/game.pxi:366, in pygambit.gambit.Game.to_arrays()
    364     array[profile] = dtype(self[profile][player])
    365 except (ValueError, TypeError, IndexError, KeyError):
--> 366     raise ValueError(
    367         f"Payoff '{self[profile][player]}' cannot be \
    368         converted to requested type '{dtype}'"

ValueError: Payoff '1' cannot be \
                        converted to requested type '<class 'dict'>'
```

You should (probably) never have a line continuation backslash inside a string.  In Python you can simply close the string and start one on the next line, and they are automatically catenated.   Further, in an exception message, you probably don't want to enforce a line break in a particular place, because where a line break should happen (if at all) would depend on the width of the user's terminal.